### ISSUE TITLE
Create screenshots for feature introduction

### ISSUE THREAD

**Issue by bugless-it** [fit-together-tv/ft-issue-report#10]
@nagthanm 
Our new landing page is https://hello.chuongnau.com.

If you look at the "feature introduction" section (the one with 6 cards), you will see something like:

----- logo -----
         title
    description

For each of the feature, can you help take a screenshot of the related functionality? What you can do is:

- Go to the screen where that functionality can be used - e.g. rich-text editor -> go to create question form.
- Perform some tasks on the app in order to showcase the feature - e.g. create a new question, add some rich text.
- Record a short video and take a screenshot of the first frame of the video.

The design I want to achieve is each of the feature card has a screenshot, and once the user hovers mouse over a card, the video will play for that specific card.

**Comment by nagthanm** [comment]
Hi, Could you suggest a video editor software I can use? Many thanks.

**Comment by bugless-it** [comment]
@nagthanm for screen capturing, you can use the xbox game bar on windows.
For video editor software, most importantly you need to be able to crop the videos. Unless you have a macbook?

**Comment by nagthanm** [comment]
hi, which file type should be recommended for edit.

**Comment by nagthanm** [comment]
https://user-images.githubusercontent.com/87641533/128621248-7cbd3e70-0d7b-4b18-9f2e-04de60f6be1e.mp4

**Comment by bugless-it** [comment]
@nagthanm I'd suggest a full screen capture of the app, going from one flow to another:
- Create, edit and save question form.
- Create and start tests.
- Take part in test in incognito mode, submit answers.
- Manage submissions in view submission tab.

Then we can cut the video into multiple parts.

**Comment by nagthanm** [comment]
Hi, is a video like this OK?
 Chú Ong Nâu - Google Chrome 2021-08-10 11-30-54...
<https://drive.google.com/file/d/1ysrGX90mi4qEsf_5tIBbtbQC49NCkK_Z/view?usp=drive_web>

**Comment by bugless-it** [comment]
@nagthanm yes i think this is good. Could you keep the same flow, but make some changes:
- Change browser language setting to Vietnamese so you can use the app in Vietnamese?
- Type more quickly for title/description/questions.
Other than that, the flow is good.

**Comment by nagthanm** [comment]
Hi, it seems like Xbox Game Bar can not record Chrome in incognito mode. 
@bugless-it

**Comment by nagthanm** [comment]
The same is happened in Microsoft Edge.

**Comment by nagthanm** [comment]
That is so weird, I can record and sent you videos in issue #13

**Comment by nagthanm** [comment]
Oh ok, I need to be in Incognito mode first so the video can be recorded.

**Comment by nagthanm** [comment]
Hi, here's the videos for:
+ Create, edit and save question form.
+ Create and start tests.
+ Take part in test in incognito mode, submit answers.

https://drive.google.com/file/d/1obRHMvl9jXmGJhU-LO1IopNIbZqLLLJz/view?usp=sharing

Could you solve the #12 issue then I can do the task "Manage submissions in view submission tab."

**Comment by bugless-it** [comment]
@nagthanm I've commented on #12.

**Comment by nagthanm** [comment]
I made a video for "Manage submissions in view submission tab." and uploaded to youtube.
Please tell me if we need more improvement on these videos.

https://www.youtube.com/playlist?list=PLzzbcDv2nP-y8PXBCk-1x_XSpIuwDcmQq&jct=cYvoAu8aJ2dA1Ic9XQB9r5GK80rS_Q
